packages/quantic/cypress/e2e/default-2/smart-snippet/smart-snippet.cypress.ts

```diff
@@ -79,159 +92,291 @@ describe('quantic-smart-snippet', {browser: 'chrome'}, () => {
           });
         });
       });
+      analyticsModeTest.forEach((analytics) => {
+        describe(analytics.label, () => {
+          before(() => {
+            analyticsMode = analytics.mode;
+          });
 
-      describe('when the query returns a smart snippet', () => {
-        describe('when the smart snippet answer is not collapsed', () => {
-          it('should properly display the smart snippet', () => {
-            visitPage({useCase: param.useCase});
-
-            scope('when loading the page', () => {
-              Expect.displaySmartSnippetCard(true);
-              Expect.displaySmartSnippetQuestion(exampleSmartSnippetQuestion);
-              Expect.displaySmartSnippetAnswer({
-                text: exampleAnswerText,
-                link: {
-                  text: exampleInlineLinkText,
-                  href: exampleInlineLinkUrl,
-                },
-              });
-              Expect.displaySmartSnippetSourceUri(exampleSmartSnippetSourceUri);
-              Expect.displaySmartSnippetSourceTitle(
-                exampleSmartSnippetSourceTitle
-              );
-              Expect.displayCollapsedSmartSnippetAnswer(false);
-              Expect.displayExpandedSmartSnippetAnswer(false);
-              Expect.displaySmartSnippetAnswerToggle(false);
-            });
+          describe('when the query returns a smart snippet', () => {
+            describe('when the smart snippet answer is not collapsed', () => {
+              it('should properly display the smart snippet', () => {
+                visitPage({useCase: param.useCase});
 
-            scope('when the source title is clicked', () => {
-              Actions.clickSmartSnippetSourceTitle();
-              Expect.logOpenSmartSnippetSource({
-                title: exampleSmartSnippetSourceTitle,
-                uri: exampleSmartSnippetSourceUri,
-                permanentId: examplePermanentId,
-              });
-            });
+                scope('when loading the page', () => {
+                  Expect.displaySmartSnippetCard(true);
+                  Expect.displaySmartSnippetQuestion(
+                    exampleSmartSnippetQuestion
+                  );
+                  Expect.displaySmartSnippetAnswer({
+                    text: exampleAnswerText,
+                    link: {
+                      text: exampleInlineLinkText,
+                      href: `${location.origin}/examples/${pageUrl}#`,
```

Comment:
Why is this changing?
Seems weird that the location.origin is used to determine the href of the SS source

Reply:
Before this change, we were using a random url as the SS source, it was for example `https://www.coveo.com/en`.

The problem with that, is first I noticed some flakiness when doing that originating from an internal cypress error that sometimes occurs when from a page you navigate to another page and then you come back to your page, this problem was rarely occurring tho, so that's not the main reason of doing this.

The main reason is that I wanted to simplify the tests and avoiding navigating to multiple pages, so I wanted to make the SS source points towards `#`  so when we click on it we will stay on the same page.

when we set the herf of the ss source to `#`, `location.origin` is automatically set as the herf to make the link navigate to the same page and that's why I'm using it here.